ibis/expr/types/relations.py

```diff
@@ -1722,30 +1722,58 @@ def select(
 
     projection = select
 
+    @util.deprecated(
+        as_of="7.0",
+        instead=(
+            "use `Table.rename` instead (if passing a mapping, note the meaning "
+            "of keys and values are swapped in Table.rename)."
+        ),
+    )
     def relabel(
         self,
         substitutions: Mapping[str, str]
         | Callable[[str], str | None]
         | str
         | Literal["snake_case", "ALL_CAPS"],
+    ) -> Table:
+        """Deprecated in favor of `Table.rename`"""
+        if isinstance(substitutions, Mapping):
+            substitutions = {new: old for old, new in substitutions.items()}
+        return self.rename(substitutions)
+
+    def rename(
+        self,
+        method: str
+        | Callable[[str], str | None]
+        | Literal["snake_case", "ALL_CAPS"]
+        | Mapping[str, str]
+        | None = None,
+        /,
+        **substitutions: str,
     ) -> Table:
         """Rename columns in the table.
 
         Parameters
         ----------
+        method
+            An optional method for renaming columns. May be one of:
+
+            - A format string to use to rename all columns, like
+              ``"prefix_{name}"``.
+            - A function from old name to new name.
+            - The literal strings ``"snake_case"`` or ``"ALL_CAPS"`` to
```

Comment:
total nit: "The literal string" not "The literal strings"

Reply:
There are two literal strings here, having it plural seems more grammatically correct to me.